tests/SprykerEcoTest/Zed/Adyen/Business/FacadeTest.php

```diff
@@ -228,6 +228,28 @@ public function testHandleOnlineTransferResponseFromAdyen(): void
         }
     }
 
+    /**
+     * @return void
+     */
+    public function testHandleOnlineTransferResponseFromAdyenAfterNotification(): void
+    {
+        $facade = $this->createFacade();
```

Comment:
Please, add `// Arrange // Act // Assert` comments. 
Check for example `Bundles/StockAddress/tests/SprykerTest/Zed/StockAddress/Business/StockAddressFacadeTest.php`